mobile/library/cc/engine_builder.h

```diff
@@ -244,6 +245,7 @@ class EngineBuilder {
   std::string http3_connection_options_ = "";
   std::string http3_client_connection_options_ = "";
   std::vector<std::pair<std::string, int>> quic_hints_;
+  std::vector<std::string> quic_suffixes_;
```

Comment:
nit: consider adding "canonical" to the name of this member to match the method.